src/functions/instance/state/describeTableCache.js

```diff
@@ -0,0 +1,23 @@
+const descriptionCache = new Map(JSON.parse(sessionStorage.getItem('descriptionCache')) || []);
```

Comment:
JSON.parse will fail if `sessionStorage.getItem('descriptionCache')` is invalid or `undefined`. If its `null` it will return `null`. I think you should try to make this safer. 

Maybe a function that wraps it in a try/catch: `() =>  { try { JSON.parse... } catch { return [] } }`

Reply:
How about this?
```javascript
const descriptionCache = () => new Map(JSON.parse(sessionStorage.descriptionCache ?? '{}'));
```